docs/contributing.md

```diff
@@ -1,9 +1,11 @@
 ## Contributions policy
 
-We are currently not accepting public contributions (but we do plan to in later releases and we’ll keep you posted on this). However, we are taking issues and we do want your [feedback]({{urlRoot}}/index#give-us-feedback).
+We are currently not accepting public contributions (but we do plan to in later releases and we’ll keep you posted on this). However, we are taking issues and we do want your feedback. Please come and talk to us about the software and the documentation via:
+
+  * **The SpatialOS forums** - Visit the [support section](https://forums.improbable.io/c/sup) in our forums and use the unity-gdk tag.
```

Comment:
"https://forums.improbable.io/c/sup" -> 'sup, how's it with you?

Code marker on `unity-gdk` tag?

Reply:
https://forums.improbable.io/latest?tags=unity-gdk